.github/dependabot.yml

```
@@ -5,6 +5,7 @@ updates:
   schedule:
     interval: daily
     time: "10:00"
+  target_branch: "staging"
```

```suggestion
  target-branch: "staging"
```

`dependabot`'s docs suggest that this should be `target-branch` instead: https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#target-branch